exdb/templates/exdb/home.html

```diff
@@ -0,0 +1,61 @@
+{% extends "exdb/base.html" %}
+{% load i18n %}
+{% load staticfiles %}
+{% block css %}
+    {{ block.super }}
+    <link rel="stylesheet" type="text/css" href="{% static 'exdb/css/home.css' %}"/>
+{% endblock %}
+{% block content %}
+    {{ block.super }}
+    <h3 class="log-out"><a href="{% url 'logout' %}">{% trans "Log out" %}</a></h3>
+    <h2>{% trans 'Hello' %} {{ user }}</h2>
+    <p><a href={% url 'create_experience' %}>{% trans 'Create an experience' %}</a></p>
+    <hr class="break" />
+    <div class="experience-panel">
+        {% for status, experiences in experience_dict.items %}
+            {% if experiences|length %}
+                <div class='experience-div'>
+                    <h1 class="status">{{ status }}</h1>
+                    {% for ex in experiences %}
+                        <div class="experience">
+                            <a href=
+                                "{% if ex.needs_evaluation %}
+                                    {% url 'conclusion' pk=ex.pk %}
+                                {% elif ex.next_approver_id == user.pk and ex.status == 'pe' %}
```

Comment:
I would like to see this decision consolidated somewhere so that we do not have the opportunity for inconsistency if the conditions for what constitutes an approval changes. The best way that I can think of would be to create a method which determines when someone can approve an experience then either store a boolean on the experience passed to the model or create a template tag to call the method with the required arguments.